src/sbt-test/kotlin/basic-tests/test

```diff
@@ -0,0 +1,3 @@
+> compile
+> test
+> "test-only *SimpleTest"
```

Comment:
It would be great if you could also verify that `target/test-reports/SimpleTest.xml` and `target/test-reports/demo.JavaCalculator.xml` gets created with expected contents

Reply:
Done!